content/en/docs/instrumentation/net/traces-api.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Tracing API reference
+linkTitle: .NET Tracing API
```

Comment:
To make these links more visually consistent with other language SIGs, how about this for a short title:
```suggestion
linkTitle: API - tracing
```